### DISCUSSION TITLE
Possible trademark violation

### DISCUSSION THREAD

**Discussion by ascheja** [provectus/kafka-ui#1468] [General]
Disclaimer: I'm not associated with The Apache Foundation in any way.

Hi there,

The Apache Foundation is quite strict with its trademarks and as such will come by in time with the demand to remove "Kafka" from the project name. Some references:
- https://github.com/edenhill/kcat/pull/339
- https://github.com/yahoo/CMAK/issues/713

Just wanted to give a heads-up before you run into any trouble.

**Comment by Haarolean** [comment]
Hi, thanks for your concern. 

We're well aware and have discussed this with Apache before. That's why you can notice the explicit use of "Apache Kafka" term.
